.github/workflows/deploy_valyent.yml

```
@@ -0,0 +1,64 @@
+  FLEET_NAME: message-board-fleet
+  VALYENT_PROJECT: message-board
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Mock Valyent CLI
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2016:info:3:6: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)